src/doc/src/reference/config.md

```diff
@@ -401,27 +399,6 @@ schedule overlapping invocations of `rustc` in parallel when possible.
 
 The `[cargo-new]` table defines defaults for the [`cargo new`] command.
 
-##### `cargo-new.name`
```

Comment:
Can you keep these headings, and instead mark them as deprecated like `target.<triple>.ar`?

Reply:
Done 👍🏻